ui/footer.py

```
@@ -1,17 +1,52 @@
+import subprocess
 import streamlit as st
 from datetime import datetime
 
 
+def get_version():
+    try:
+        version = (
+            subprocess.check_output(["git", "describe", "--tags"], stderr=subprocess.STDOUT)
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)